localizations/zh_CN.json

```
@@ -268,103 +268,103 @@
     "Conditionals": "条件",
     "Conditioning Highres.fix strength (for sd-v1-5-inpainting)": "高清修复原图调节强度 (专为 sd-v1-5-inpainting 设计)",
     "Config Presets": "预设配置",
-    "Config file for Adapter models": "自适应模型(Adapter models)的配置文件",
-    "Config file for Control Net models": "ControlNet 模型的配置文件",
+    "Config file for Adapter models": "Adapter 模型配置文件路径",
+    "Config file for Control Net models": "ControlNet 模型配置文件路径",
     "Config-Presets": "预设配置",
     "Containing directory": "目标模型目录",
     "ControlNet-M2M": "ControlNet-视频转绘",
-    "Converted checkpoints will be saved in your": "转化换后的模型(checkpoint)将会保存在",
+    "Converted checkpoints will be saved in your": "转换出的模型将会保存在你的",
     "Copy Metadata": "开始复制元数据",
-    "Copy config from": "从...复制配置文件",
-    "Copy image to:": "将图片复制到:",
-    "Copy metadata to other models in directory": "将元数据复制到其他目录中的模型",
+    "Copy config from": "复制配置文件",
+    "Copy image to:": "复制当前图像到: ",
+    "Copy metadata to other models in directory": "将元数据复制到文件夹中的其他模型",
```

```suggestion
    "Copy metadata to other models in directory": "将元数据复制到目录中的其他模型",
```
Directory - 目录  
Folder - 文件夹